client.go

```diff
@@ -248,6 +267,25 @@ func (client *Client) MultiRangeWithOptions(fromTimestamp int64, toTimestamp int
 	return
 }
 
+// MultiReverseRangeWithOptions - Query a timestamp range across multiple time-series by filters, in reverse direction.
+// args:
+// fromTimestamp - start of range. You can use TimeRangeMinimum to express the minimum possible timestamp.
+// toTimestamp - end of range. You can use TimeRangeFull or TimeRangeMaximum to express the maximum possible timestamp.
+// mrangeOptions - MultiRangeOptions options. You can use the default DefaultMultiRangeOptions
+// filters - list of filters e.g. "a=bb", "b!=aa"
+func (client *Client) MultiReverseRangeWithOptions(fromTimestamp int64, toTimestamp int64, mrangeOptions MultiRangeOptions, filters ...string) (ranges []Range, err error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
```

Comment:
@filipecosta90 same here, don't you feel you can avoid this code duplication?

Reply:
I would strict the duplicate of code ( that is little ) due to the reasons above...

Reply:
fixed in last commit :+1: